rss-reader/Application/AppDelegate.m

```diff
@@ -0,0 +1,40 @@
+//
+//  AppDelegate.m
+//  rss-reader
+//
+//  Created by Uladzislau on 11/17/20.
+//
+
+#import "AppDelegate.h"
+#import "FeedRouter.h"
+
+@interface AppDelegate ()
+
+@property (nonatomic, retain) FeedRouter* router;
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    UIWindow *window = [[UIWindow alloc] initWithFrame:UIScreen.mainScreen.bounds];
```

Comment:
You are creating too many local variables. No need to spare additional memory. Please, use the following signature:
self.window = [[UIWindow alloc] initWithFrame:UIScreen.mainScreen.bounds];
self.router = [[FeedRouter alloc] initWithWindow:window];